README.md

```diff
@@ -1,3 +1,5 @@
+# HasteFuzz: sbft-2023 placeholder
```

Comment:
Sorry but you need to add your fuzzer in a sub-directory under [`fuzzers/`](https://github.com/google/fuzzbench/tree/master/fuzzers) and not change any existing file.
Here is [our guide to add fuzzers](https://google.github.io/fuzzbench/getting-started/adding-a-new-fuzzer/).

It has to be this way because we will merge your PR into the master branch to run experiments. Changing any other file will overwrite our FuzzBench repo.

Reply:
Yes I will push our true commit later